contracts/lib/LibPrices.sol

```diff
@@ -65,7 +65,14 @@ library Prices {
 
         for (uint256 i = 0; i < n; i++) {
             PriceInstant memory currPrice = prices[i];
-            averagePrices[i] = averagePrice(currPrice);
+            uint256 _averagePrice = averagePrice(currPrice);
+
+            // dont inclue periods that have no trades
+            if (_averagePrice == 0 && currPrice.trades == 0) {
+                continue;
+            } else {
+                averagePrices[i] = averagePrice(currPrice);
+            }
```

Comment:
I think the if... else... is redundant here because in both the cases _averagePrice == 0 && currPrice.trades == 0, averagePrice(...) would have returned 0:

    https://github.com/tracer-protocol/tracer-protocol/blob/contracts-v2/contracts/lib/LibPrices.sol#L49

Reply:
This was a bit broken before; fixed now so that it ignores periods where there's zero trades in the mean